src/locales/ar/LC_MESSAGES/messages.po

```diff
@@ -0,0 +1,623 @@
+msgid ""
+"Git is a type of version control system (VCS) that makes it easier to "
+"track changes to files. "
+msgstr ""
+" Git  هو نوع من أنواع نظم التحكم في الإصدارات  (VCS)  يُسهل عليك تعقب "
+"التغييرات على الملفات "
+
+#: services/messenger.py:382 services/messenger.py:723
+msgid ""
+"For example, when you edit a file, Git can help you determine exactly "
+"what changed, who changed it, and why."
+msgstr ""
+" على سبيل المثال ، عندما تقوم بتحرير ملف ، يمكن أن يساعدك  Git  في تحديد "
+"ما تغير بالضبط ومن قام بتغييره ولماذا. "
+
+#: services/messenger.py:391 services/messenger.py:732
+msgid "👶🏽 Install Git"
+msgstr " 👶🏽   تثبيت  Git "
+
+#: services/messenger.py:394 services/messenger.py:734
+msgid "🤓 I've Git Installed"
+msgstr " 🤓  لقد قمت بتثبيت  Git "
+
+#: services/messenger.py:398 services/messenger.py:738
+msgid "Want to learn more about Git?"
+msgstr " هل تريد معرفة المزيد عن  Git  ؟ "
+
+#: services/messenger.py:411
+msgid "Good decision 👌🏽"
+msgstr " قرار جيد  🏽👌 "
+
+#: services/messenger.py:415
+msgid ""
+"We are going to split the process into 5 steps: \n"
+"🛵 Fork, Clone, Update, Push and Merge. "
+msgstr ""
+" سنقسم العملية إلى 5 خطوات:  \n"
+" 🛵 تفريع، استنساخ، تحديث، دفع ودمج. "
+
+#: services/messenger.py:424
+msgid "🥢 1. Fork"
+msgstr " ١) التفريع 🥢 "
+
+#: services/messenger.py:427
+msgid "Ready for the first step?!"
+msgstr "جاهز للخطوة الأولى  ؟! "
+
+#: services/messenger.py:435
+msgid "Awesome 👌🏽"
+msgstr " ممتاز  👌🏽 "
+
+#: services/messenger.py:439
+msgid ""
+"Open this link in a new window: \n"
+"https://github.com/fbdevelopercircles/open-source-edu-bot"
+msgstr ""
+" افتح هذا الرابط في نافذة جديدة:  \n"
+" https://github.com/fbdevelopercircles/open-source-edu-bot"
+
+#: services/messenger.py:446
+msgid "Now, click `Fork` on the top right corner of your screen"
+msgstr "الآن ، انقر فوق \"تفريع\" في الزاوية اليمنى العليا من شاشتك"
+
+#: services/messenger.py:456
+msgid "A copy of the original project will be created in your account."
+msgstr " سيتم إنشاء نسخة من المشروع الأصلي في حسابك "
+
+#: services/messenger.py:464
+msgid "🚃 2. Clone"
+msgstr " ٢) الاستنساخ 🚃 "
+
+#: services/messenger.py:467
+msgid "Ready for the next step?!"
+msgstr "مستعد للخطوة التالية ؟! "
+
+#: services/messenger.py:475
+msgid "Great 👌🏽"
+msgstr " عظيم 👌🏽"
+
+#: services/messenger.py:479
+msgid ""
+"Right now, you have a fork of the `open-source-edu-bot` repository, but "
+"you don't have the files in that repository on your computer."
+msgstr ""
+" الآن ، لديك تفرع من مستودع  `open-source-edu-bot`  ، لكن ليس لديك ملفات "
+"هذا المستودع على جهاز الكمبيوتر الخاص بك. "
+
+#: services/messenger.py:487
+msgid ""
+"Let's create a clone of your fork locally on your computer.\n"
+"On GitHub, in your newly forked project, Click on `Code` above the list "
+"of files"
+msgstr ""
+" لنقم بإنشاء نسخة من تفريعتك على جهاز الكمبيوتر الخاص بك.  \n"
+"  على  GitHup   ، في مشروعك الجديد المتفرع ، انقر فوق \"الرمز\" أعلى جدول"
```

Comment:
you should keep `Code`, as the image contains it...

Reply:
Done